packages/ant/build.sh

```diff
@@ -5,22 +5,16 @@ TERMUX_PKG_SRCURL=http://apache.mirrors.spacedump.net//ant/binaries/apache-ant-$
 TERMUX_PKG_SHA256=ffc6c9eb2d153db76e18df70fa145744ddd0d754339df5a797a47603cfeaa8a6
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_RECOMMENDS="openjdk-17"
+TERMUX_PKG_DEPENDS="openjdk-17"
 TERMUX_PKG_PLATFORM_INDEPENDENT=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
 termux_step_make_install() {
-	mkdir -p $TERMUX_PREFIX/share/ant/lib
-
-	for jar in ant ant-launcher; do
-		$TERMUX_D8 \
-		--classpath $ANDROID_HOME/platforms/android-$TERMUX_PKG_API_LEVEL/android.jar \
-		--release \
-		--min-api $TERMUX_PKG_API_LEVEL \
-		--output $TERMUX_PREFIX/share/ant/lib/${jar}.jar \
-			lib/${jar}.jar
-	done
-
-	install $TERMUX_PKG_BUILDER_DIR/ant $TERMUX_PREFIX/bin/ant
-	perl -p -i -e "s%\@TERMUX_PREFIX\@%${TERMUX_PREFIX}%g" $TERMUX_PREFIX/bin/ant
+        mv ./bin/ant .
+	rm -f ./bin/*
+        mv ./ant ./bin/
+	rm -rf $TERMUX_PREFIX/opt/ant
+	mkdir -p $TERMUX_PREFIX/opt/ant
+	cp -r ./* $TERMUX_PREFIX/opt/ant
+        ln -sfr $TERMUX_PREFIX/opt/ant/bin/ant $PREFIX/bin/ant
```

Comment:
Inconsistent indentation and make sure you're using tabs.

Reply:
My bad, poorly edited from github editor in the browser. Just fixed it